admin_guide/managing_projects.adoc

```diff
@@ -45,7 +45,7 @@ $ oadm create-bootstrap-project-template -o yaml > template.yaml
 . Modify the template by adding objects or modifying existing objects:
 +
 ----
-$ oc edit template template.yaml
+$ vim template.yaml
```

Comment:
How about instead of suggesting an editor to use, we drop the command example and just modify the text say:

```
Use a text editor to modify the *_template.yaml_* file by adding objects or modifying existing objects.
```